resolwe/flow/execution_engines/python/__init__.py

```diff
@@ -2,11 +2,10 @@
 import inspect
 import os
 import shutil
-from importlib import import_module
 
 from resolwe.flow.execution_engines.base import BaseExecutionEngine
-from resolwe.process.runtimes import python_runtimes_manager
 from resolwe.process.parser import SafeParser
+from resolwe.process.runtimes import python_runtimes_manager
```

Comment:
How is this connected to the "Copy all runtimes to executor" commit message?

Reply:
It should be merged with the previous commit with the similar name.